src/main/java/in/twizmwaz/cardinal/util/Vectors.java

```diff
@@ -66,4 +66,18 @@ public static Vector getVector(String str) {
     return coordinates;
   }
 
+  public static Vector min() {
+    return new Vector(Double.NEGATIVE_INFINITY, Double.NEGATIVE_INFINITY, Double.NEGATIVE_INFINITY);
+  }
+
+  public static Vector max() {
+    return new Vector(Double.POSITIVE_INFINITY, Double.POSITIVE_INFINITY, Double.POSITIVE_INFINITY);
+  }
+
+  public static Vector getMirroredVector(Vector vector, Vector origin, Vector normal) {
+    vector = vector.minus(origin);
+    Vector projected = vector.minus(normal.times(vector.getCrossProduct(normal).multiply(2)));
+    return projected.add(origin);
```

Comment:
I'm not sure if this function actually works, but i think it should, any good way of testing this?